animal-tracker-express/server.js

```
@@ -0,0 +1,45 @@
+// import path from 'path';
+const port = 3005;
+// import express from 'express';
+const express = require('express');
+// import compression from 'compression';
+const app = express();
+app.use(express.json());
+const pgp = require('pg-promise')();
+// import pgp from 'pg-promise'
+const db = pgp('postgres://postgres@localhost:5432/animal-tracker')
+
+
+app.get('/sightings', (req, res) => {
+    db.any('SELECT individuals.id, individuals.species, sightings.common_name, individuals.nickname, sightings.location, sightings.ishealthy, sightings.sighter_email, sightings.record_timestamp FROM individuals INNER JOIN sightings ON individuals.id = sightings.animal_id')
+    .then(data => {
+     res.status(200).send(data);
+    }).catch(function(error) {
+        res.send(error);
+      });
```

It's unusual to send errors to the client and if you did want to you should just send the message. Did this even log what you expected?

Since we're in control of the server and client, I would avoid explicitly handling errors and letting Express catch them and turn it into a 500. Add logging only if the default handling hides the error from you in the Express log.